app/views/static/alert-overview-popover.html.haml

```diff
@@ -0,0 +1,10 @@
+%div
+  .tip-title
+    {{item.name}}
+  .tip-row{"ng-repeat" => "alert in vm.hoverAlerts track by $index"}
+    .tip-col.host-col{"ng-if" => "$index < 5"}
```

Comment:
Aaah.. it's OK I think .. if it's about limiting the number of alerts to display, it makes sense :)

(You'll still get the `.tip-row` for each, but if it doesn't matter.. :)) (If it does, you *can* use ng-if on the row itself)